config/main.py

```diff
@@ -337,6 +340,20 @@ def run_command(command, display_cmd=False, ignore_error=False):
     if proc.returncode != 0 and not ignore_error:
         sys.exit(proc.returncode)
 
+def _get_device_type():
```

Comment:
This is a good candidate to move to sonic-py-common

Reply:
will add.